mlfinlab/sampling/bootstrapping.py

```diff
@@ -13,14 +14,16 @@ def get_ind_matrix(triple_barrier_events, price_bars):
     how to form it. We decided that using triple_barrier_events and price bars by analogy with concurrency
     is the best option.
 
-    :param triple_barrier_events: (pd.DataFrame): triple barrier events from labeling.get_events
+    :param triple_barrier_events: (pd.Series): triple barrier events(t1) from labeling.get_events
```

Comment:
Could you please rename `triple_barrier_events` to `samples_info_sets` in the `sampling` module as well ?

Reply:
Done